src/lightning_app/utilities/network.py

```diff
@@ -27,10 +27,10 @@ def find_free_network_port() -> int:
     return port
 
 
-_CONNECTION_RETRY_TOTAL = 5
+_CONNECTION_RETRY_TOTAL = 2880
```

Comment:
isn't it too long?

Reply:
Probably! do you see any potential issues because of a larger retry?